MailCore/Utils/MessageBodyUtils.swift

```diff
@@ -44,6 +45,32 @@ public enum MessageBodyUtils {
         "blockquote[type=\"cite\"]"
     ]
 
+    public static func prepareWithPrintOption(message: Message) async -> PresentableBody? {
+        guard let messageBody = message.body else {
+            return nil
+        }
+        let bodyValue = messageBody.value ?? ""
+
+        do {
+            let printHeader = try createPrintHeader(message: message)
+            let originalDocument = try await SwiftSoup.parse(bodyValue).insertChildren(0, [printHeader])
+
+            let bodyFromDoc = try originalDocument.outerHtml()
+
+            let messageBodyQuote = await splitBodyAndQuote(messageBody: bodyFromDoc)
+
+            return PresentableBody(
+                body: messageBody,
+                compactBody: messageBodyQuote.messageBody,
+                quote: messageBodyQuote.quote
+            )
+        } catch {
+            print("error: \(error.localizedDescription)")
```

Comment:
If we cannot prepare the body with print options, do we want to notify the user with a snackbar?